MessageBird/Net/RestClient.cs

```diff
@@ -234,16 +235,16 @@ public virtual string PerformHttpRequest(string method, string uri, string body,
             }
         }
 
-        public virtual string PerformHttpRequest(string method, string uri, HttpStatusCode expectedStatusCode)
+        public virtual string PerformHttpRequest(string method, string baseUrl, string uri, HttpStatusCode expectedStatusCode)
         {
             string body = null;
 
-            return PerformHttpRequest(method, uri, body, expectedStatusCode);
+            return PerformHttpRequest(method, baseUrl, uri, body, expectedStatusCode);
         }
 
-        private HttpWebRequest PrepareRequest(string method, string requestUriString)
+        private HttpWebRequest PrepareRequest(string method, string requestUriString, string endpoint)
```

Comment:
Maybe swap the order of the params so `endpoint` comes first? I think this is a bit confusing as I would logically expect that part to come first.